subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -48,5 +128,95 @@ static int init_reset(void)
 		 sizeof(cig_pool) / sizeof(struct ll_conn_iso_group),
 		 &cig_free);
 
+	for (int h = 0; h < CONFIG_BT_CTLR_CONN_ISO_GROUPS; h++) {
+		ll_conn_iso_group_get(h)->cig_id = 0xFF;
+	}
+
+	/* Initialize LLL */
+	err = lll_conn_iso_init();
+	if (err) {
+		return err;
+	}
+
 	return 0;
```

Comment:
suggest using define for numeric constant 0

Reply:
Returning hard 0 is de facto standard for "success" in Zephyr. Error codes have enums/defines.